api/envoy/extensions/bootstrap/internal_listener/v3/internal_listener.proto

```diff
@@ -19,4 +22,9 @@ option (xds.annotations.v3.file_status).work_in_progress = true;
 
 // Configuration for internal listener.
 message InternalListener {
+  // The internal listener client connection buffer size in KB.
+  // For example, if buffer_size is set to 5, then the actual buffer size is
+  // 5 KB = 5 * 1024 bytes.
+  // If the buffer_size is not specified, the buffer size is set to 1024 KB.
+  google.protobuf.UInt32Value buffer_size = 1 [(validate.rules).uint32 = {lte: 8192}];
```

Comment:
What will be the result of explicitly setting this to 0? will it be an infinite buffer size?

Reply:
adding the lower bound to be 1k.